### ISSUE TITLE
[adwaita_icon_theme] [x86_64} Unable to build

### ISSUE THREAD

**Issue by supechicken666** [skycocker/chromebrew#4527]
Output:
```shell
done
for file in `cd ../../Adwaita/scalable; find . -name "*.svg"`; do \
        context="`dirname $file`"; \
        /bin/mkdir -p /usr/local/tmp/crew/dest/usr/local/share/icons/Adwaita/scalable/$context; \
        /bin/bash /usr/local/tmp/crew/adwaita-icon-theme-3.38.0.tar.xz.dir/adwaita-icon-theme-3.38.0/install-sh -c -m 644 ../../Adwaita/scalable/$file /usr/local/tmp/crew/dest/usr/local/share/icons/Adwaita/scalable/$file; \
        for size in 16x16 24x24 32x32 48x48 64x64 96x96; do \
                /bin/mkdir -p /usr/local/tmp/crew/dest/usr/local/share/icons/Adwaita/$size/$context; \
                /usr/local/bin/gtk-encode-symbolic-svg ../../Adwaita/scalable/$file $size -o /usr/local/tmp/crew/dest/usr/local/share/icons/Adwaita/$size/$context; \
        done \
done
Can't load file: Unrecognized image file format
Can't load file: Unrecognized image file format
Can't load file: Unrecognized image file format
Can't load file: Unrecognized image file format
Can't load file: Unrecognized image file format
Can't load file: Unrecognized image file format
Can't load file: Unrecognized image file format
```
It loop forever on my chromium os laptop

**Comment by satmandu** [comment]
Yup. this is a huge pain because you get a bunch of dependencies around loading svg files. Hence the need to make sure that librsvg loads correctly first, and that gdk_pixbuf has the librsvg support as well.

Really the librsvg needs to be compiled first, and then gdk_pixbuf, and one also has to make sure that gdk_pixbuf has its cache filled correctly. We also may need to make the user stuff at the end of the gdk_pixbuf install not optional. (the stuff involving putting stuff into .bashrc.)

**Comment by uberhacker** [comment]
Fixed in #4552.
